remoteworkitem/flatten_test.go

```diff
@@ -0,0 +1,37 @@
+package remoteworkitem_test
+
+import (
+	"encoding/json"
+	"testing"
+
+	"github.com/almighty/almighty-core/remoteworkitem"
+	"github.com/almighty/almighty-core/resource"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestFlattenMap(t *testing.T) {
+	resource.Require(t, resource.UnitTest)
+	testString := []byte(`{"admins":[{"name":"aslak"}],"name":"shoubhik", "assignee":{"fixes": 2, "complete" : true,"foo":[ 1,2,3,4],"1":"sbose","2":"pranav","participants":{"4":"sbose56","5":"sbose78"}},"name":"shoubhik"}`)
```

Comment:
Then you would manually have to create the map the way you 'think' the json parser would create it.